<Feature: Playtest World>

projojoboy opened the pull request.

Commit:
<Feature: Villager Path Finding>
Add Idle State
Add Idle Actions
Add Wander State
Add Wander Actions
Add Timer Decisions
Add Destination Reached Decisions
Add WalkTowards State
Add Test Scene
Add Material Red
Add Material Blue
Add Material Green
Add Material Gray

Commit:
Merge branch 'development' into feature/villager-path-finding

# Conflicts:
#	Assets/Scripts/Features.meta

Commit:
<Feature: Villager Path Finding>
Update pull request, changes request

Commit:
<Feature: Villager Path Finding>
Add AI to main scene
Add Little tweaks to make the AI movement veel better
Add New map structure

empireboy committed:
<Feature: Villager Path Finding>
Add Gizmo for targetLocation

Commit:
Merge branch 'development' into feature/villager-path-finding

Commit:
<Feature: Villager Path Finding>
Undo scene changes

Commit:
<Feature: Playtest World>
Add Prefab playtest world

empireboy reviewed (changes requested):
Has to be implemented in the GameScene first

projojoboy committed:
Merge branch 'development' into feature/playtest-world

# Conflicts:
#	Assets/Prefabs.meta
#	Assets/Scenes/GameScene.unity

projojoboy committed:
<Feature: Playtest World>
Add World prefab to game scene
Add Villager models
Add Camera settings

empireboy committed:
<Feature: Playtest World>

Fix Removed meta files
Fix Change VillagerAnimator file location

empireboy committed:
<Feature: Playtest World>

Fix Folder names
Fix Placed command scripts an a commands folder
Fix Camera dragging speed in GameScene

empireboy committed:
<Feature: Playtest World>

Fix Changed file location for AI scripts

empireboy reviewed (approved):
Has to be implemented in the GameScene first
